build.sbt

```diff
@@ -33,22 +33,20 @@ libraryDependencies += "org.slf4j" % "slf4j-api" % slf4jVersion
 libraryDependencies += "io.lemonlabs" %% "scala-uri" % scalaUriVersion
 
 //Testing
-libraryDependencies += "org.scalatest" %% "scalatest" % scalaTestVersion % "test"
-libraryDependencies += "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test"
-libraryDependencies += "org.scalamock" %% "scalamock-scalatest-support" % scalaMockVersion % "test"
-libraryDependencies += "ch.qos.logback" % "logback-classic" % logbackVersion % "test"
+libraryDependencies += "org.scalatest" %% "scalatest" % scalaTestVersion % Test
+libraryDependencies += "org.scalacheck" %% "scalacheck" % scalacheckVersion % Test
+libraryDependencies += "org.scalamock" %% "scalamock" % scalaMockVersion % Test
+libraryDependencies += "ch.qos.logback" % "logback-classic" % logbackVersion % Test
 
 scalacOptions ++= Seq(
   "-deprecation",
-  "-Xfatal-warnings",
+  // "-Xfatal-warnings",
```

Comment:
Why is this commented?

Reply:
Indeed, we want to keep it. This took me more time than expected. See https://github.com/algolia/algoliasearch-client-scala/pull/571/commits/e86a20b3a888df227dd24bb54c3a9490b93487a1 cc @Ant-hem @dragisak

Reply:
You get warning because `scala.collection.JavaConverters` are deprecated in Scala 2.13